src/module/vue/components/mce-editor.vue

```diff
@@ -12,18 +17,98 @@
       v-html="$enrichHtml(modelValue)"
     />
     <div v-else class="editor-content" v-html="$enrichHtml(modelValue)"></div>
-    <a class="editor-edit">
-      <i class="fas fa-edit" @click="data.editing = true"></i>
-    </a>
-  </div>
+    <BtnFaicon
+      class="editor-edit btn-block"
+      icon="fas fa-edit"
+      @click="data.editing = true"
+    />
+  </article>
 </template>
 
+<style lang="less">
+@import '../../../styles/forms.less';
+@import '../../../styles/button.less';
+.mce-editor {
+  height: 100%;
+  min-height: 100px;
+  color: var(--ironsworn-color-fg);
+  background-color: initial;
+
+  .editor-content {
+    height: inherit;
+    min-height: inherit;
+  }
+  .editor-edit {
+    background-color: var(--ironsworn-color-btn-bg-enabled);
```

Comment:
Looks like this is turning the background gray. Is this intentional? This seems to go counter to the Foundry default and the norm of the other systems I tried. I'm not sure I like it.

dnd5e in Foundry v9:
![dnd5e](https://user-images.githubusercontent.com/39902/196039683-6e735869-f1ba-4ffd-9452-3b5245490884.jpg)

Blades in the dark, which is so minimal I assume it's the default Foundry styles:
![Blades in the Dark](https://user-images.githubusercontent.com/39902/196039764-bef9d36c-ece6-4b3c-a83f-20ec753056dc.jpg)

And 13th Age, which actually gets lighter in edit mode:
![13th age](https://user-images.githubusercontent.com/39902/196039749-2467af4a-2cb9-41e3-910d-b3ff0428d0a6.jpg)

Reply:
the idea is to provide a visual cue for "this is a (rich) text input box" that has some relationship to other (plain) text input boxes, at least when the field is editable.

ultimately, tho, harmonizing that might require more aggressive restyling of the editor component than is practical to include in this PR (and are inessential to its purpose, anyways)